apps/rush-lib/src/rush.schema.json

```diff
@@ -149,5 +164,53 @@
     }
   },
   "additionalProperties": false,
-  "required": [ "npmVersion", "rushMinimumVersion", "projects" ]
+  "required": [ "npmVersion", "rushMinimumVersion", "projects" ],
+  "definitions": {
+    "lockStepVersion": {
+      "type":"object",
+      "description": "Lockstep version policy",
+      "properties": {
+        "policyName": {
+          "description": "The name of the version policy",
+          "type": "string"
+        },
+        "baseType": {
+          "description": "The name of base version policy",
+          "type": "string",
+          "enum": [ "lockStepVersion" ]
+        },
+        "version": {
+          "description": "Current version for projects with lockStepVersion policy",
+          "type":"string"
+        },
+        "nextBump": {
+          "description": "Type of next version bump",
+          "enum": [ "prerelease", "release", "minor", "patch", "major" ]
+        }
```

Comment:
This setting describes an implementation detail of an algorithm.  In order for a person to choose the right value, they need to understand the algorithm correctly.

It might be better to configure the scenario rather than the algorithm, e.g. something like this:

branchType=master  --> bumps are always NPM prerelease tag
branchType=release  --> bumps are always patch-level